ci-jobs/functional/run_appium.yml

```diff
@@ -5,6 +5,9 @@ steps:
   displayName: Install Node 11.x
 - script: npm install -g appium@beta --chromedriver_version='2.44'
   displayName: Install appium
+- script: npm install -g opencv4nodejs
```

Comment:
how much longer the tests take now?

Reply:
This installation step takes 8m46s.
In total, it's 20min.
https://dev.azure.com/ki4070ma/python-client/_build/results?buildId=1046&view=logs

Reply:
what was the previous value (before enabling opencv)?

Reply:
It's 10m30s previously.
Almost just added installation time.
https://dev.azure.com/ki4070ma/python-client/_build/results?buildId=1047

Reply:
I'm ok with the change if such increase in tests execution duration works for you